core/services/keeper/validate_test.go

```diff
@@ -32,12 +32,14 @@ func TestValidatedKeeperSpec(t *testing.T) {
 		{
 			name: "valid job spec",
 			args: args{
-				tomlString: testspecs.GenerateKeeperSpec(testspecs.KeeperSpecParams{
```

Comment:
nit. delete his function?

Ot it might also be useful to keep using this function and just remove the obs source param

Reply:
You mean remove the observation source from the toml string? It's removed already, unless I misunderstood your question :)

Reply:
I meant the function `testspecs.GenerateKeeperSpec`